spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/couchbase/CouchbaseAutoConfigurationIntegrationTests.java

```diff
@@ -74,7 +74,7 @@ static void createBucket() {
 	void setUp() {
 		this.context = new AnnotationConfigApplicationContext();
 		this.context.register(CouchbaseAutoConfiguration.class);
-		TestPropertyValues.of("spring.couchbase.bootstrap-hosts=localhost",
+		TestPropertyValues.of("spring.couchbase.bootstrap-hosts=" + couchbase.getContainerIpAddress(),
```

Comment:
FYI Testcontainers 1.14.2 adds `getHost()` (returns the same value as `getContainerIpAddress()` but less confusing), consider using it once Spring Boot updates TC to 1.14.2